content/en/network_monitoring/devices/netflow.md

```diff
@@ -83,15 +85,126 @@ From the **Configuration** tab in NetFlow, click **Add Enrichment** to upload th
-
-This data is also available in Dashboards, Notebooks, and more for more precise queries and for correlating with other sources of data.
-{{< img src="network_device_monitoring/netflow/notebook.png" alt="Notebook" width="80%" >}}
+You can access the data collected by NetFlow Monitoring on the [**NetFlow** page][5]. Hover over a flow from the list for additional information about hosts, pods, and containers, and access related network connections.
+
+{{< img src="network_device_monitoring/netflow/information.png" alt="Hover over a flow aggregated from a device emitting netflow to access related network connections" width="100%" >}}
+
+When creating a [NetFlow monitor][6], you should consider the following fields with respect to the source IP or destination IP from the perspective of the device. These fields provide insights into network traffic patterns and help with optimizing performance and security.
+
+### Interface Information
+
+Represents details about the ingress and egress interfaces.
+
+| Field Name | Field Description |
+|---|---|
+| Egress Interface Alias | Alias of the egress interface. |
+| Egress Interface Index | Index of the egress interface. |
+| Egress Interface Name | Name of the egress interface. |
+| Ingress Interface Alias | Alias of the ingress interface. |
+| Ingress Interface Index | Index of the ingress interface. |
+| Ingress Interface Name | Name of the ingress interface. |
+
+### Device Information
+
+Represents details related to the device generating NetFlow records.
+
+| Field Name | Field Description |
+|---|---|
+| Device IP | IP address used to map to a device in NDM for enrichment purposes. |
+| Exporter IP | IP address from which NetFlow packets originate. |
+| Device Model | Model of the device. |
+| Device Name | Name of the device. |
+| Device Namespace | Namespace of the device. |
+| Device Vendor | Vendor of the device. |
+
+### Flow Details
+
+Represents characteristics of the network flow.
+
+| Field Name | Field Description |
+|---|---|
+| Direction | Indicates whether the flow is inbound or outbound. |
+| Start Time | Timestamp of the first network packet between the source and destination IP addresses. |
+| End Time | Timestamp of the last network packet between the source and destination IP addresses. |
+| Ether Type | Type of Ethernet frame encapsulation (IPv4 or IPv6). |
+| Flow Type | Type of NetFlow data format (IPFIX, sFlow5, NetFlow5, NetFlow9, or Unknown). |
+| IP Protocol | Protocol used for communication (such as ICMP, TCP, or UDP). |
+| Next Hop IP | IP address of the next hop in the network path. |
+| TCP Flag | Union of all TCP flags observed over the life of the flow. |
+| Bytes | Total number of bytes transferred. |
+| Packets | Total number of packets transferred. |
+
+Use the out-of-the-box facets to start analyzing traffic patterns based on NetFlow destination and source IP addresses.
```

Comment:
Is this line in the right place? Should it be under a facets-related section? When placed here, it looks like it pertains to the Flow Details section

Reply:
```suggestion
In addition to fields, you can also use out-of-the-box facets to start analyzing traffic patterns based on NetFlow destination and source IP addresses.
```